gocourse05/server/zoo/cameras/camera_name1.go

```diff
@@ -0,0 +1,26 @@
+package cameras
+
+import (
+	"gocourse05/server"
+	cameras "gocourse05/server/zoo/photo"
```

Comment:
Не впевнений, що для пакету `photo` треба аліас `cameras`.
```suggestion
	"gocourse05/server/zoo/photo"
```